compatibility/versions/UpdateVersions.hs

```diff
@@ -162,11 +153,6 @@ getChecksums ver = do
                "https://raw.githubusercontent.com/digital-asset/daml/v" <> SemVer.toString ver
                <> "/templates/create-daml-app-test-resources/messaging.patch"
           | otherwise = Nothing
-        mbSandboxOnXUrl
```

Comment:
don't we need to apply the check 2.0.0 <= x < 2.7.0

Reply:
Scratch that, we are still using it for old platform versions in conformance tests.

Reply:
253c6746fd8a493d75509c1a0db9d1d534dff53c